zou/app/blueprints/assets/resources.py

```diff
@@ -139,6 +139,9 @@ def get(self):
             criterions["assigned_to"] = persons_service.get_current_user()[
                 "id"
             ]
+            criterions[
```

Comment:
It should not be added if the user is not a vendor.

Reply:
It's the case there's permissions.has_vendor_permissions() line 138